### PR TITLE
Doc: Update example for current version

### PR DESCRIPTION
**Pull request by vjravi**
Update example. Fix new line errors on webpage and update example imports

cc @dynamicwebpaige

### CODE REVIEW

File: tensorflow/python/keras/backend.py [hunk 1]
```diff
@@ -1344,21 +1344,19 @@ def cast(x, dtype):
...
-      # It doesn't work in-place as below.
-      >>> K.cast(input, dtype='float16')
-      <tf.Tensor 'Cast_1:0' shape=(2, 3) dtype=float16>
-      >>> input
-      <tf.Tensor 'Placeholder_2:0' shape=(2, 3) dtype=float32>
-      # you need to assign it.
-      >>> input = K.cast(input, dtype='float16')
-      >>> input
-      <tf.Tensor 'Cast_2:0' shape=(2, 3) dtype=float16>
+      >>> import tensorflow as tf
```
Comment (dynamicwebpaige) [inline tensorflow/python/keras/backend.py]: No need to use `K` as a proxy for `tf.keras.backend` in this case, and please don't use `input` as a variable name. :slightly_smiling_face: Otherwise, LGTM! Please make the mod, and I'll approve the commit.

```python
import tensorflow as tf

test = tf.keras.backend.ones(shape=(1, 3))
print(test)

cast_test = tf.keras.backend.cast(test, dtype='float64')
print(cast_test)
```

**Review by dynamicwebpaige — CHANGES_REQUESTED** [review]
See requested fixes inline. Thanks for your contribution! :slightly_smiling_face:

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by harman-de-asr** [comment]
Hi @tensorflowbutler I shall resume work on this shortly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
